linter.py

```diff
@@ -62,10 +62,8 @@ def find_errors(self, output):
             for warning in data['warnings']:
                 line = warning['line'] - self.line_col_base[0]
                 col = warning['column'] - self.line_col_base[1]
-                type = warning['severity']
-                text = warning['text']
+                severity = warning['severity']
+                rule = warning['rule']
+                text = warning['text'].replace('(' + rule + ')', '')
 
-                if type == 'warning':
-                    yield (warning, line, col, "", type, text, None)
-                else:
-                    yield (warning, line, col, type, "", text, None)
+                yield (warning, line, col, rule, severity, text, None)
```

Comment:
The tuple thing is discouraged and soon deprecated bc it is basically unreadable. It maybe does what you want but the new style is

```
yield LintMatch(
	match=warning,
	line=line,
	col=col,
	error_type=severity,
	code=rule,
	message=text
)
```

I think we forgot to export `LintMatch` so you have to do

```
from SublimeLinter.lint.linter import LintMatch
```